components/button/button.css

```diff
@@ -0,0 +1,101 @@
+button{
```

Comment:
please avoid using direct element targeting! you can use class instead.

Reply:
Noted.